buildpipeline/pipeline.json

```diff
@@ -74,6 +74,20 @@
             "Platform": "arm",
             "Type": "build/product/"
           }
+        },
```

Comment:
This file is for official builds, not for the CI. The CI is defined in the .groovy files in the same folder.
We cannot enable official builds of corefx until official build of coreclr produces nuget packages and uploads them to myget.

Reply:
Understood.  Creating PR keeps it in my queue..

Reply:
@janvorli Were you suggesting I split this PR? Can corefx CI be enabled before coreclr official builds?